### DISCUSSION TITLE
Inserir data atual no Visualg

### DISCUSSION THREAD

**Discussion by tinaquintela** [frontendbr/forum#2224] [Perguntas e Respostas]
Estou começando a aprender programação e gostaria muito se alguém em ajudasse com essa dúvida.
Só estou querendo saber se tem como inserir data atual do sistema no Visualg, se alguém puder me ajudar, ficaria agradecida.

**Comment by DavyFranz** [comment]
Oi, também estou começando pelo Visualg, creio que tu esteja aprendendo com os vídeos do Guanabara. Tô querendo montar um grupo no WhatsApp pra juntar o pessoal que esteja começando pra nós trocar uma ideia, tirar dúvidas e se ajudar nos projetos. Se tiver interesse eu vou deixar o link do grupo aqui.

https://chat.whatsapp.com/GPsrTlqE24KLAQ4982ZpDW
